apiextensions/storageversion/migrator.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/kelseyhightower/envconfig"
```

Comment:
envconfig seems overweight for just looking up a single env var 

It's probably simpler to use `os.Getenv` and `strconv.ParseBool`

Reply:
Sure. Added e1741aed98464514cdf2d55b7a592eed6624ebc3